scripts/build.ps1

```diff
@@ -122,6 +122,7 @@ $env:MSBUILD_VERSION = "15.0"
 Write-Verbose "Setup build configuration."
 $TPB_Solution = "TestPlatform.sln"
 $TPB_TestAssets_Solution = Join-Path $env:TP_ROOT_DIR "test\TestAssets\TestAssets.sln"
+$TPB_TestAssets_CILAssets = Join-Path $env:TP_ROOT_DIR "test\TestAssets\CILProject\CILProject.proj"
```

Comment:
Why can't this be in the test assets solution as any other test project?

Reply:
It's an IL project which Visual Studio does not support, so I can't add it into the `TestAssets.sln`.